modules/migration-editing-pvs-in-migplan.adoc

```diff
@@ -35,11 +49,15 @@ spec:
       name: mysql
       namespace: mysql-persistent
     selection:
-      action: copy
-      copyMethod: filesystem
-      storageClass: cephrbd <1>
-      accessMode: ReadWriteMany <2>
+      action: <copy> <1>
+      copyMethod: <filesystem> <2>
+      verify: <true> <3>
+      storageClass: <gp2> <4>
+      accessMode: <ReadWriteMany> <5>
     storageClass: cephfs
 ----
-<1> Destination storage class.
-<2> Destination access mode.
+<1> Allowed values are `move`, `copy`, and `skip`. If only one action is supported, the default value is the supported action. If multiple actions are supported, the default value is `copy`.
+<2> Allowed values are `snapshot` and `filesystem`. Default value is `filesystem`.
+<3> For file system copy method only: Set to `true` to verify the files after they are copied. Default value is `null`.
+<4> Allowed values are listed in the `status.destStorageClasses` block of the `MigPlan` CR. The value must match the `name` field of one of the `status.destStorageClasses` values. If this value is not specified, a default storage class is selected.
```

Comment:
An empty value is also valid here. If a value is not specified, then the resulting destination PVC will have no storage class specified.